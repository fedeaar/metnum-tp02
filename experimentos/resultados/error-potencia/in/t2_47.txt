7.455560000000000000e+06 1.706732000000000000e+06 9.655680000000000000e+05 4.060610000000000000e+05 -2.180858000000000000e+06 -2.900720000000000000e+05 -7.539210000000000000e+05 5.147020000000000000e+05 -3.000870000000000000e+05 1.876464000000000000e+06 2.473000000000000000e+05 4.612200000000000000e+04 2.905420000000000000e+05 1.465837000000000000e+06 1.993222000000000000e+06 -2.917196000000000000e+06 -1.680317000000000000e+06 6.507910000000000000e+05 -1.054988000000000000e+06 -1.137020000000000000e+05
1.706732000000000000e+06 7.228536000000000000e+06 8.767920000000000000e+05 1.375412000000000000e+06 -1.045618000000000000e+06 -1.548840000000000000e+05 6.883200000000000000e+05 2.938550000000000000e+06 -1.318938000000000000e+06 -1.139410000000000000e+06 5.039370000000000000e+05 4.354430000000000000e+05 2.305946000000000000e+06 9.047900000000000000e+05 -9.288680000000000000e+05 -9.272470000000000000e+05 -2.533065000000000000e+06 -2.310283000000000000e+06 1.251129000000000000e+06 2.055155000000000000e+06
9.655680000000000000e+05 8.767920000000000000e+05 5.610628000000000000e+06 2.175979000000000000e+06 -9.712400000000000000e+04 1.143789000000000000e+06 -2.903106000000000000e+06 -6.925700000000000000e+04 -2.158404000000000000e+06 5.677740000000000000e+05 -2.713990000000000000e+05 -4.406780000000000000e+05 1.010742000000000000e+06 9.357270000000000000e+05 -6.725460000000000000e+05 -1.203700000000000000e+06 5.478810000000000000e+05 7.159800000000000000e+05 -2.789669000000000000e+06 5.496530000000000000e+05
4.060610000000000000e+05 1.375412000000000000e+06 2.175979000000000000e+06 6.508080000000000000e+06 -7.971010000000000000e+05 4.337360000000000000e+05 -5.922340000000000000e+05 -4.472700000000000000e+05 -4.138640000000000000e+05 -1.518193000000000000e+06 -7.728040000000000000e+05 1.442700000000000000e+06 1.261000000000000000e+06 1.079378000000000000e+06 -1.534636000000000000e+06 -2.299453000000000000e+06 2.032121000000000000e+06 -8.738500000000000000e+04 1.744500000000000000e+05 3.819373000000000000e+06
-2.180858000000000000e+06 -1.045618000000000000e+06 -9.712400000000000000e+04 -7.971010000000000000e+05 6.312068000000000000e+06 2.321700000000000000e+04 1.072942000000000000e+06 -6.330730000000000000e+05 -6.567260000000000000e+05 -4.200482000000000000e+06 -9.189500000000000000e+04 -1.264715000000000000e+06 -7.221270000000000000e+05 -2.297860000000000000e+05 -2.600274000000000000e+06 7.655860000000000000e+05 1.916518000000000000e+06 1.867598000000000000e+06 -1.271306000000000000e+06 -1.572465000000000000e+06
-2.900720000000000000e+05 -1.548840000000000000e+05 1.143789000000000000e+06 4.337360000000000000e+05 2.321700000000000000e+04 4.139181000000000000e+06 -2.631218000000000000e+06 -1.859270000000000000e+05 1.188624000000000000e+06 -1.107065000000000000e+06 1.178581000000000000e+06 7.585080000000000000e+05 -7.373200000000000000e+05 9.596290000000000000e+05 -2.040760000000000000e+05 -3.238010000000000000e+05 8.862990000000000000e+05 6.440170000000000000e+05 -2.329076000000000000e+06 1.034149000000000000e+06
-7.539210000000000000e+05 6.883200000000000000e+05 -2.903106000000000000e+06 -5.922340000000000000e+05 1.072942000000000000e+06 -2.631218000000000000e+06 6.410610000000000000e+06 1.215999000000000000e+06 -6.444890000000000000e+05 -5.622380000000000000e+05 1.049340000000000000e+05 9.905990000000000000e+05 6.228330000000000000e+05 -3.208740000000000000e+05 9.676400000000000000e+05 7.406320000000000000e+05 -3.298930000000000000e+05 -1.654443000000000000e+06 3.781263000000000000e+06 -4.851020000000000000e+05
5.147020000000000000e+05 2.938550000000000000e+06 -6.925700000000000000e+04 -4.472700000000000000e+05 -6.330730000000000000e+05 -1.859270000000000000e+05 1.215999000000000000e+06 7.588107000000000000e+06 -3.304100000000000000e+04 6.752780000000000000e+05 -5.820940000000000000e+05 1.857861000000000000e+06 2.556256000000000000e+06 2.552900000000000000e+06 -1.601680000000000000e+06 3.402115000000000000e+06 -2.899020000000000000e+06 -1.290050000000000000e+06 2.874107000000000000e+06 6.555050000000000000e+05
-3.000870000000000000e+05 -1.318938000000000000e+06 -2.158404000000000000e+06 -4.138640000000000000e+05 -6.567260000000000000e+05 1.188624000000000000e+06 -6.444890000000000000e+05 -3.304100000000000000e+04 7.868945000000000000e+06 5.595780000000000000e+05 -5.916400000000000000e+05 3.048675000000000000e+06 2.887050000000000000e+05 3.213230000000000000e+05 1.813961000000000000e+06 1.295113000000000000e+06 -1.584841000000000000e+06 1.340282000000000000e+06 -5.368800000000000000e+04 1.175710000000000000e+06
1.876464000000000000e+06 -1.139410000000000000e+06 5.677740000000000000e+05 -1.518193000000000000e+06 -4.200482000000000000e+06 -1.107065000000000000e+06 -5.622380000000000000e+05 6.752780000000000000e+05 5.595780000000000000e+05 7.989170000000000000e+06 -1.773595000000000000e+06 1.416470000000000000e+05 1.675900000000000000e+05 -8.569730000000000000e+05 2.078174000000000000e+06 6.059040000000000000e+05 -9.814950000000000000e+05 -1.926496000000000000e+06 1.234757000000000000e+06 -2.050090000000000000e+05
2.473000000000000000e+05 5.039370000000000000e+05 -2.713990000000000000e+05 -7.728040000000000000e+05 -9.189500000000000000e+04 1.178581000000000000e+06 1.049340000000000000e+05 -5.820940000000000000e+05 -5.916400000000000000e+05 -1.773595000000000000e+06 6.196185000000000000e+06 3.188300000000000000e+05 2.355420000000000000e+05 1.811586000000000000e+06 1.502449000000000000e+06 -3.144590000000000000e+06 -2.725243000000000000e+06 3.104950000000000000e+05 -1.444041000000000000e+06 6.624870000000000000e+05
4.612200000000000000e+04 4.354430000000000000e+05 -4.406780000000000000e+05 1.442700000000000000e+06 -1.264715000000000000e+06 7.585080000000000000e+05 9.905990000000000000e+05 1.857861000000000000e+06 3.048675000000000000e+06 1.416470000000000000e+05 3.188300000000000000e+05 4.469656000000000000e+06 1.383406000000000000e+06 -2.696740000000000000e+05 -4.818000000000000000e+03 7.215510000000000000e+05 -9.096650000000000000e+05 -2.181340000000000000e+05 2.101911000000000000e+06 9.156910000000000000e+05
2.905420000000000000e+05 2.305946000000000000e+06 1.010742000000000000e+06 1.261000000000000000e+06 -7.221270000000000000e+05 -7.373200000000000000e+05 6.228330000000000000e+05 2.556256000000000000e+06 2.887050000000000000e+05 1.675900000000000000e+05 2.355420000000000000e+05 1.383406000000000000e+06 5.027636000000000000e+06 1.946357000000000000e+06 -8.969080000000000000e+05 6.496540000000000000e+05 -1.798608000000000000e+06 -1.418945000000000000e+06 -5.717610000000000000e+05 2.161243000000000000e+06
1.465837000000000000e+06 9.047900000000000000e+05 9.357270000000000000e+05 1.079378000000000000e+06 -2.297860000000000000e+05 9.596290000000000000e+05 -3.208740000000000000e+05 2.552900000000000000e+06 3.213230000000000000e+05 -8.569730000000000000e+05 1.811586000000000000e+06 -2.696740000000000000e+05 1.946357000000000000e+06 5.804627000000000000e+06 -3.388000000000000000e+05 -2.270679000000000000e+06 -2.186890000000000000e+06 8.254750000000000000e+05 -1.921856000000000000e+06 1.695331000000000000e+06
1.993222000000000000e+06 -9.288680000000000000e+05 -6.725460000000000000e+05 -1.534636000000000000e+06 -2.600274000000000000e+06 -2.040760000000000000e+05 9.676400000000000000e+05 -1.601680000000000000e+06 1.813961000000000000e+06 2.078174000000000000e+06 1.502449000000000000e+06 -4.818000000000000000e+03 -8.969080000000000000e+05 -3.388000000000000000e+05 6.339387000000000000e+06 -7.107050000000000000e+05 -1.393794000000000000e+06 7.335250000000000000e+05 -4.775670000000000000e+05 1.045331000000000000e+06
-2.917196000000000000e+06 -9.272470000000000000e+05 -1.203700000000000000e+06 -2.299453000000000000e+06 7.655860000000000000e+05 -3.238010000000000000e+05 7.406320000000000000e+05 3.402115000000000000e+06 1.295113000000000000e+06 6.059040000000000000e+05 -3.144590000000000000e+06 7.215510000000000000e+05 6.496540000000000000e+05 -2.270679000000000000e+06 -7.107050000000000000e+05 7.046168000000000000e+06 6.436480000000000000e+05 -8.720410000000000000e+05 2.118487000000000000e+06 -1.237767000000000000e+06
-1.680317000000000000e+06 -2.533065000000000000e+06 5.478810000000000000e+05 2.032121000000000000e+06 1.916518000000000000e+06 8.862990000000000000e+05 -3.298930000000000000e+05 -2.899020000000000000e+06 -1.584841000000000000e+06 -9.814950000000000000e+05 -2.725243000000000000e+06 -9.096650000000000000e+05 -1.798608000000000000e+06 -2.186890000000000000e+06 -1.393794000000000000e+06 6.436480000000000000e+05 7.138859000000000000e+06 1.297230000000000000e+05 -1.635677000000000000e+06 -1.668580000000000000e+05
6.507910000000000000e+05 -2.310283000000000000e+06 7.159800000000000000e+05 -8.738500000000000000e+04 1.867598000000000000e+06 6.440170000000000000e+05 -1.654443000000000000e+06 -1.290050000000000000e+06 1.340282000000000000e+06 -1.926496000000000000e+06 3.104950000000000000e+05 -2.181340000000000000e+05 -1.418945000000000000e+06 8.254750000000000000e+05 7.335250000000000000e+05 -8.720410000000000000e+05 1.297230000000000000e+05 5.316957000000000000e+06 -1.456987000000000000e+06 -6.987370000000000000e+05
-1.054988000000000000e+06 1.251129000000000000e+06 -2.789669000000000000e+06 1.744500000000000000e+05 -1.271306000000000000e+06 -2.329076000000000000e+06 3.781263000000000000e+06 2.874107000000000000e+06 -5.368800000000000000e+04 1.234757000000000000e+06 -1.444041000000000000e+06 2.101911000000000000e+06 -5.717610000000000000e+05 -1.921856000000000000e+06 -4.775670000000000000e+05 2.118487000000000000e+06 -1.635677000000000000e+06 -1.456987000000000000e+06 7.270802000000000000e+06 6.284950000000000000e+05
-1.137020000000000000e+05 2.055155000000000000e+06 5.496530000000000000e+05 3.819373000000000000e+06 -1.572465000000000000e+06 1.034149000000000000e+06 -4.851020000000000000e+05 6.555050000000000000e+05 1.175710000000000000e+06 -2.050090000000000000e+05 6.624870000000000000e+05 9.156910000000000000e+05 2.161243000000000000e+06 1.695331000000000000e+06 1.045331000000000000e+06 -1.237767000000000000e+06 -1.668580000000000000e+05 -6.987370000000000000e+05 6.284950000000000000e+05 7.562025000000000000e+06
