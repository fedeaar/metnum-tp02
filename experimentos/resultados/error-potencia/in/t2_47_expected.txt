2.082267726045025513e+07
1.951955731793872640e+07
1.564967164976310171e+07
1.249917836315022781e+07
1.212242638492848538e+07
1.072823436490307562e+07
6.901143218030137941e+06
5.697095491278375499e+06
5.054371101407611743e+06
4.656075694035654888e+06
4.129373321796244010e+06
4.032784855434702244e+06
2.388448337137502152e+06
2.238360927101117093e+06
1.210763421753328526e+06
9.138863792078922270e+05
4.594700109176891274e+05
2.237584513550706906e+05
3.575044962595953257e+04
1.599997848032217007e+02
